unity-renderer/Assets/DCLPlugins/ECS7/Systems/ScenesUiSystem/ECSScenesUiSystem.cs

```diff
@@ -18,6 +18,8 @@ public class ECSScenesUiSystem : IDisposable
         private readonly BaseList<IParcelScene> loadedScenes;
         private readonly BaseVariable<bool> loadingHudVisibleVariable;
 
+        private static HashSet<IParcelScene> scenesToSort = new HashSet<IParcelScene>();
```

Comment:
Please add comments why this change was required to prevent it from rolling back. Comment-less changes are fragile

Reply:
Because it allocates memory at runtime causing memory fragmentation, increased memory usage, garbage collection stalls and hiccups

Reply:
What comment are you expecting?

Reply:
Assuming there is a serial killer on the team, you need to prevent them to say "ugh, why is this static? I'll move it to a class property, it is not compatible with our standards" and then they introduce a regression in the allocations and want to come after you

Then something like `// This is a static property because ...`

Reply:
I wouldn't make that property as static, just part of the class instance state should be fine,
plus we avoid any "static property controversy" :)

Reply:
I agree with @pbosio it shouldn't be a static anyway!